test/Pester/PlatyPs.Tests.ps1

```diff
@@ -78,15 +85,17 @@ Describe 'New-PlatyPsCab' {
 
 Describe 'Format-PlatyPsHelpXml' {
 
-    $Destination = "$outFolder\CabTesting\OutXml\"
+    $Destination = "$outFolder\CabTesting\OutXml2\"
```

Comment:
I need to clean this up. At one point I was using the artifacts, however, I no longer need them. Let me adjust. 

Your point is well made, I didn't think of it.